data-source/json/test_radio_open.json

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
This should be named something like `test_radio_detail_answer_open.json` as it's specific to the detail answer in a radio being open rather than all of the radio options.

Reply:
I changed it to test_radio_detail_answer_open